src/components/themes/Themes.jsx

```diff
@@ -1,13 +1,13 @@
 import themesData from "../../data/themesData";
-
+import './Themes.css'
 export default function Theme() {
   return (
-    <section class="">
+    <section class="grid">
       {themesData.map((el) => (
-        <div class="">
-          <img src={el.themeImageURL} />
+        <div class="theme_item">
+          <div className="icon">{el.themeImageURL}</div>
           <br />
-          <div class="">{el.themeName}</div>
+          <div class=""><strong style={{fontSize:"larger"}}>{el.themeName}</strong></div>
```

Comment:
Please setup `prettier` in your preferred text editor and use `Format on save` so that code is automatically formatted before committing. Otherwise on different computers, the code is formatted separately, which creates unnecessary changes in git history.